elasticdl/python/common/model_handler.py

```diff
@@ -21,21 +24,21 @@ def get_model_to_train(self, model):
         pass
 
     @abc.abstractmethod
```

Comment:
Add some docstrings for these abstract methods so developers knows how to implement the subclasses without diving into the code for base class.